pkg/bigint/bigint_test.go

```diff
@@ -0,0 +1,24 @@
+package bigint_test
+
+import (
+	"encoding/json"
+	"github.com/ethersphere/bee/pkg/bigint"
+	"reflect"
+	"testing"
+)
+
+type TestStruct struct {
+	Bg *bigint.BigInt `json:"bg"`
```

Comment:
since this type is used only within the test, you could inline the struct definition directly within `Marshal`:
```go
mar, err := json.Marshal(struct{
Bg *bigint.BigInt
}{ 
Bg: bigint.New(5)
})